### PR TITLE
Improve user management documentation

### PR DESCRIPTION
**Pull request by laurenz**
The existing documentation was inaccurate and confusing.

This tries to address #121.

### CODE REVIEW

**Comment by 3manuek** [comment]
@nasbyj @susanmdouglas-aws @rcv-aws -- could you please review this PR?

**Review by rcv-aws — DISMISSED** [review]
not double-checking the changes, just approving

File: _usage/security.md [hunk 1]
```diff
@@ -45,9 +45,7 @@ The table below shows how Babelfish behaves for each combination:
 
 ### User management and roles
 
-By default, Babelfish comes with extra users on top of what PostgreSQL
-has to offer. Here is a list of users to be found in a fresh Babelfish
-installation: 
+Babelfis creates a number of PostgreSQL roles:
```
Comment (susanmdouglas-aws) [inline _usage/security.md:48]: Typo: Babelfish

File: _usage/security.md [hunk 2]
```diff
@@ -57,21 +55,18 @@ installation:
...
 
-These users are MS SQL-specific and try to mimic what the target database is
-supposed to look like. 
+Moreover, there will be a <code>*dbname*&lowbar;db&lowbar;owner</code>
+and a <code>*dbname*&lowbar;dbo</code> for every database you create in
+Babelfish.
 
-In addition to that, all users that are normally available in PostgreSQL are
-available as well:
+These roles are there to implement the SQL Server ownership structure.
```
Comment (susanmdouglas-aws) [inline _usage/security.md:62]: These roles implement the SQL Server ownership structure.

File: _usage/security.md [hunk 3]
```diff
@@ -57,21 +55,18 @@ installation:
...
-- `pg_monitor`
-- `pg_read_all_settings`
-- `pg_read_all_stats`
-- `pg_stat_scan_tables`
-- `pg_read_server_files`
-- `pg_write_server_files`
-- `pg_execute_server_program`
-- `pg_signal_backend`
+You can use `CREATE LOGIN` to create a new Babelfish login.  Babelfish logins
+are implemented as PostgreSQL login roles of the same name.
```
Comment (susanmdouglas-aws) [inline _usage/security.md:67]: Any login you create is automatically a user in all databases.

File: _usage/security.md [hunk 4]
```diff
@@ -57,21 +55,18 @@ installation:
...
 
-These users are unchanged, and can be used just like in a normal PostgreSQL
-installation.
+Note that Babelfish [does not support](/docs/usage/limitations-of-babelfish)
+`CREATE USER`, and any login you create is automatically a user in all
+databases.  `CREATE ROLE` is also not supported, so you cannot use roles in
+Babelfish.
```
Comment (susanmdouglas-aws) [inline _usage/security.md:72]: After moving the sentence about login behavior up, I'd simplify the last of the content:

Note that Babelfish [does not support](/docs/usage/limitations-of-babelfish)
`CREATE USER` or `CREATE ROLE` syntax.

File: _usage/security.md (hunk 1 above)
Comment (laurenz) [reply _usage/security.md]: Oops. Fixed in 66c1e14bbb6027cf6bda77acf5c6e66779dfda5d.

File: _usage/security.md (hunk 2 above)
Comment (laurenz) [reply _usage/security.md]: Yes, that is better. Done in 66c1e14bbb6027cf6bda77acf5c6e66779dfda5d.

File: _usage/security.md (hunk 3 above)
Comment (laurenz) [reply _usage/security.md:67]: Done.

File: _usage/security.md (hunk 4 above)
Comment (laurenz) [reply _usage/security.md]: Done.

**Review by rcv-aws — APPROVED** [review]
not double-checking the changes/edits, just approving

**Review by susanmdouglas-aws — APPROVED** [review]

**Pull request merged**